File: sportsbook-app/player-service/src/test/java/com/sportsbook/playerservice/model/SampleEntityTest.java
package com.sportsbook.playerservice.model;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class SampleEntityTest {

    @Test
    void testSampleEntityFields() {
        SampleEntity entity = new SampleEntity();
        entity.setId("user123");
        entity.setName("Alice");

        assertEquals("user123", entity.getId());
        assertEquals("Alice", entity.getName());
    }
}

File: sportsbook-app/player-service/src/test/kotlin/com/sportsbook/playerservice/model/SampleEntityTest.kt
package com.sportsbook.playerservice.model

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

class SampleEntityTest {

    @Test
    fun testSampleEntityFields() {
        val entity = SampleEntity()
        entity.id = "user123"
        entity.name = "Alice"

        assertEquals("user123", entity.id)
        assertEquals("Alice", entity.name)
    }
}
